crates/astria-sequencer-relayer/src/relayer.rs

```diff
@@ -41,9 +41,8 @@ pub(crate) struct Relayer {
     // The limit that relayer will pay for submitting sequencer blocks to the DA.
     gas_limit: u64,
 
-    // Carries the signing key to sign sequencer blocks before they are submitted to the data
-    // availability layer.
-    validator: Validator,
+    // If this is set, only relay blocks to DA which are proposed by the same validator key.
+    validator: Option<Validator>,
```

Comment:
How does this relate to the refactor?

Reply:
see my reply - it's to optionally keep the functionality as previously, where we only relay blocks from a certain proposer